test/e2e/examples.go

```diff
@@ -84,13 +90,13 @@ var _ = KubeDescribe("[Feature:Example]", func() {
 
 			By("checking up the services")
 			checkAllLogs := func() {
-				forEachPod(c, ns, "name", "redis", func(pod api.Pod) {
+				forEachPod("name", "redis", func(pod api.Pod) {
```

Comment:
entirely on scope.  yes.  This is a local wrapper , no need for `forEachPod` global wrapper, this fixes #23730 .